members/members.css

```diff
@@ -0,0 +1,115 @@
+.title {
+    text-align: center;
+    margin-bottom: 2em;
+    font-size: 3.5em;
+}
+
+.sub-title {
+    font-size: 25px;
+    color: #444444;
+    position: relative;
+    padding-bottom: 1.25em;
+}
+
+.sub-title:after{
+    border-bottom: solid 1px;
+    bottom: 0;
+    content: '';
+    display: inline-block;
+    left: 0;
+    position: absolute;
+    width: 3rem;
+}
+
+h1 {
+    font-size: 30px;
+}
+
+h2{
+    font-size: 25px;
+}
+
+h3{
+    font-size: 20px;
+}
+
+h4{
+    font-size: 15px;
+}
+
+/* Five columns side by side */
+.column-card {
+  float: left;
+  width: 20%;
+  padding: 0 8px;
+  margin-bottom: 16px;
+}
+
+/* Display the columns below each other instead of side by side on small screens */
+@media screen and (max-width: 650px) {
+  .column-card {
+    width: 100%;
+    display: block;
+  }
+}
+
+/* avatar label alignment */
+.container-card {
+  padding: 0 16px 16px 16px;
+  margin: 10px 0;
+  text-align: center;
+}
+
+.title-card {
+  color: grey;
+}
+
+.avatar {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  object-fit: cover;
+  border-radius:50%;
+  width: 120px;
+  max-width: 150px;
+  height: 120px;
+  cursor: pointer;
```

Comment:
It might be better to make it so that only the members with modals have additional styling (`cursor` and `opacity`). Right now, a user can hover over a member in another section (e.g. Website committee) and it looks as if you can click on their avatar even though nothing will happen. 

I'm not sure what the best approach would be tho; conditional class assignment doesn't look as neat :( 

```
            <div class="column-card">
               <div class="card">
                        <img id="{{ item.identification }}-img" src="{{ site.baseurl }}{{ item.image }}" 
                        {% if item.modal %}
                            class="avatar hoverable"
                        {% else %}
                            class="avatar"
                        {% endif %}
                        >
                        <div class="container-card">
                            <h3>{{ item.title }}</h3>
                            <p class="title-card">{{ item.description }}</p>
                        </div>
                    </div>
                </div>
```

Reply:
I changed the class name of the exec avatar so that it will have different styling from other avatars